network/src/peers/peers.rs

```diff
@@ -140,99 +85,12 @@ impl Peers {
             self.broadcast_pings().await;
 
             // Store the peer book to storage.
-            self.save_peer_book_to_storage()?;
+            // self.save_peer_book_to_storage()?;
```

Comment:
this is not the best place for the update, but we should keep it as-is during the refactoring; we can remove it from here once we implement a clean node shutdown

Reply:
fixed